exe/genv-devices

```diff
@@ -12,20 +12,32 @@ import genv
 Device = Dict[Any, Any]
 Devices = Dict[int, Device]
 
+"""
+This module manages all genv devices. Each genv device represents a single Nvidia gpu.
+A device can be attached to 0, 1 or more environments.
+"""
+
 
 def get_devices_total_memory() -> Iterable[str]:
     """
     Get total memory of all devices as string using nvidia-smi.
 
     Cached for performance purposes as this should not change.
-    Can be mocked with environment variables.
+    Can be mocked using the GENV_MOCK_DEVICE_COUNT and GENV_MOCK_DEVICE_MEMORY environment variables.
+    :return: A list of gpu devices memory.
+            The memory at index i represents the memory (in mib) of the gpu device at index i.
     """
+
     def _impl():
         if 'GENV_MOCK_DEVICE_COUNT' in os.environ:
-            return [os.environ.get('GENV_MOCK_DEVICE_MEMORY', '16g') for _ in range(int(os.environ['GENV_MOCK_DEVICE_COUNT']))]
+            return [os.environ.get('GENV_MOCK_DEVICE_MEMORY', '16g') for _ in
```

Comment:
is this `for _ in` in the right line?
it seems that it is misplaced here?

Reply:
It is a simple formating change - the continuation of the for is in the next line.
Do you want me to revert it?

Reply:
ooooh no no my bad I thought it was something else